benchmark/source_map/source-map-cache.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+const path = require('path');
+
+const options = {
+  flags: ['--expose-internals'],
+};
+
+const bench = common.createBenchmark(
+  main,
+  {
+    operation: [
+      'findSourceMap-valid',
+      'findSourceMap-generated-source',
+    ],
+    n: [1e5],
+  },
+  options,
+);
+
+function main({ operation, n }) {
+  const Module = require('node:module');
+
+  Module.setSourceMapsSupport(true);
```

Comment:
Option `generatedCode` should be true for `eval` support.

```suggestion
  Module.setSourceMapsSupport(true, {
    generatedCode: true,
  });
```